homework/hw03-[1].py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+pip install mrjob
```

Comment:
why is this here?  This breaks your code.  You should either submit the jupyter notebook as a jupyter notebook, or the python file you generated.  Not this python code that doesn't run.

-2